wtpython/display.py

```diff
@@ -68,26 +64,52 @@ def __init__(
     ) -> None:
         self.questions = questions
         super().__init__(name=name)
+        self._text: Text | None = None
```

Comment:
If I'm not mistaken, this will only allow 3.10 because allowing ``|`` in the typing system got introduced in 3.10

You might want to replace this with Union[Text, None]